cdap-operational-stats-core/src/main/java/co/cask/cdap/operations/hdfs/HDFSInfo.java

```diff
@@ -86,8 +90,13 @@ public String getWebURL() {
           }
         }
       }
-    } catch (IOException e) {
-      LOG.warn("Error in determining HDFS URL. Web URL of HDFS will not be available in HDFS operational stats.", e);
+      lastCollectFailed = false;
+    } catch (Exception e) {
+      // TODO: remove once CDAP-7887 is fixed
```

Comment:
So what is it help to do:
`hdfsConf.set(CommonConfigurationKeys.HADOOP_SECURITY_SERVICE_USER_NAME_KEY, nameNodePrincipal);`?